ITKImageProcessingFilters/ImportAxioVisionV4Montage.cpp

```diff
@@ -102,6 +103,8 @@ class ImportAxioVisionV4MontagePrivate
   QString m_MontageInformation;
   bool m_ImportAllMetaData = false;
   QString m_MetaDataAttributeMatrixName;
+  int32_t m_MaxRow = 0;
+  int32_t m_MaxCol = 0;
   std::vector<ImportAxioVisionV4Montage::BoundsType> m_BoundsCache;
```

Comment:
Added a cached value for the max number of columns and rows. Those values can be used when generating the MontageInformation string and also when sanity checking the inputs from the user.